pages/common/expr.md

```diff
@@ -3,22 +3,30 @@
 > Evaluate expressions and manipulate strings.
 > More information: <https://www.gnu.org/software/coreutils/expr>.
 
-- Get string length:
+- Get the length of a specific string:
 
 `expr length {{string}}`
 
-- Evaluate logical or math expression with an operator ('+', '-', '*', '&', '|', etc.). Special symbols should be escaped:
+- Get the substring of a string with a specific range:
 
-`expr {{first_argument}} {{operator}} {{second_argument}}`
+`expr substr {{string}} {{from}} {{length}}`
 
-- Get position of the first character in 'string' that matches 'substring':
+- Match a specific substring agains an anchored pattern:
 
-`echo $(expr index {{string}} {{substring}})`
+`expr match {{string}} {{pattern}}`
 
-- Extract part of the string:
+- Get a substring position in a specific string:
 
-`echo $(expr substr {{string}} {{position_to_start}} {{number_of_characters}}`
+`expr index {{string}} {{substring}}`
```

Comment:
This command prints the first occurrence of a char in a "collection"  of chars. This doesn't work with substring. e.g.  
```bash
$ expr index freeradius radius
2
$ expr index freeradius free
1
$ expr index freeradius dius
7
$ expr index freeradius abdf
1
```

Reply:
Fixed. :)